src/routes/about/+page.svelte

```diff
@@ -13,11 +13,11 @@
 <div class="grid grid-rows-[auto,1fr]" style="height:100vh;height:100dvh;">
     <div class="navbar">
         <Navbar>
-            <!-- <div slot="left-buttons" /> -->
-            <label for="sidebar" slot="center">
-                <div class="btn btn-ghost normal-case text-xl">{$t['Menu_About']}</div>
-            </label>
-            <!-- <div slot="right-buttons" /> -->
+            {#snippet center()}
+                <label for="sidebar">
```

Comment:
@davidmoore1: Here is an example of where there is a `<label for="sidebar">` so that if you tap/click on the about title, it shows the sidebar menu.

Positive: you can get to the menu without having to go back and then click on the hamburger menu.
Negative: It is different than the mobile apps. It is a hidden feature.

Reply:
I would say leave it in.  We don't have the swipe in capability to get the sidebar so this is a good alternative I think